22-053/sections/00-preface.adoc

```diff
@@ -2,9 +2,9 @@
 [[abstract_section]]
 == Abstract
 
-CityGML is an open conceptual data model for the storage and exchange of virtual 3D city models. It is defined through a Unified Modeling Language (UML) object model. This UML model extends the ISO Technical Committee 211 (TC211) conceptual model standards for spatial and temporal data. Building on the ISO fundation assures that the man-made features described in the City Models share the same spatial-temporal universe as the surrounding countryside within which they reside.
+POI-CM is an open conceptual data model for representing information about points of interest (POI). It is defined through a Unified Modeling Language (UML) object model. This UML model extends the ISO Technical Committee 211 (TC211) conceptual model standards for spatial and temporal data. Building on the ISO fundation assures that the features described in the POI Models share the same spatial-temporal universe as features described by related standards (e.g., CityGML).
 
-The aim of the development of CityGML is to reach a common definition of the basic entities, attributes, and relations of a 3D city model. This is especially important with respect to the cost-effective sustainable maintenance of 3D city models, allowing the reuse of the same data in different application fields.
+The aim of the development of the OGC POI standard is to reach a common definition of the basic entities, attributes, and relations of "points of interest". In the most broad terms, a point of interest is a location about which information of general interest is available. A POI can be as simple as a set of coordinates and an identifier, or more complex such as a three dimensional model of a building with names in various languages, information about open and closed hours, and a civic address
```

Comment:
The definition of POI needs to appear in the terminology definitions clause (it looks like that may need to be added?) and, if one exists, a source for the definition needs to be cited in the definition and populated in the bibliography. 

```suggestion
The aim of developing the OGC POI conceptual model is to reach a common definition of the basic entities, attributes, and relations of "points of interest." In the broadest terms, a point of interest is a location about which information of general interest is available. A POI can be as simple as a set of coordinates and an identifier, or more complex such as a three-dimensional model of a building with names in various languages, information about open and closed hours, and a civic address.
```

Reply:
You can create a new issue with the suggestion to create a terms/definitions clause or a glossary, or point to the clause in the draft standard, and then it will kick this comment down the road for future task.